### ISSUE TITLE
rqt_plot: "edit curves and line axes" does not work

### ISSUE THREAD

**Issue by ahendrix** [ros-visualization/rqt_common_plugins#103]
When I push the "edit curves and line axes" button, I get a stacktrace:

```
Traceback (most recent call last):
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/backend_qt4.py", line 474, in edit_parameters
    figureoptions.figure_edit(axes, self)
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/qt4_editor/figureoptions.py", line 132, in figure_edit
    icon=get_icon('qt4_editor_options.svg'), apply=apply_callback)
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/qt4_editor/formlayout.py", line 511, in fedit
    dialog = FormDialog(data, title, comment, icon, parent, apply)
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/qt4_editor/formlayout.py", line 427, in __init__
    self.formwidget.setup()
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/qt4_editor/formlayout.py", line 399, in setup
    widget.setup()
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/qt4_editor/formlayout.py", line 374, in setup
    widget.setup()
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/qt4_editor/formlayout.py", line 281, in setup
    field.addItems(value)
TypeError: QComboBox.addItems(QStringList): argument 1 has unexpected type 'list'
```

ros-groovy-rqt-plot 0.2.16-0
ros-groovy-python-qt-binding 0.2.9-0
Ubuntu Precise 12.04

**Comment by vincentrou** [comment]
I do not reproduce this issue on Ubuntu 12.04 with :
 ros-hydro-rqt-plot 0.3.12
 ros-hydro-python-qt-binding 0.2.16
 python-matplotlib 1.1.1
Should be closed ?

**Comment by trainman419** [comment]
Confirmed fixed in Indigo on Ubuntu 14.04 as well:
ros-indigo-rqt-plot 0.3.12
ros-indigo-python-qt-binding 0.2.17
python-matplotlib 1.3.1

**Comment by dirk-thomas** [comment]
Thank you both for checking.

I will close the ticket. If anyone is still having this problem please feel free to comment here.
